extra/changelog-generator/test-changelog-generator

```diff
@@ -399,6 +429,12 @@ _Released xx.xx.xxxx_
 
 #### test-changelog-generator.$$ (HEAD)
 
+* Aggregated Dependabot Changelogs:
+  Bump github.com/klauspost/pgzip from 1.2.3 to 1.2.4
+  Bumps [github.com/klauspost/pgzip](https://github.com/klauspost/pgzip) from 1.2.3 to 1.2.4.
+  - [Release notes](https://github.com/klauspost/pgzip/releases)
+  - [Commits](klauspost/pgzip@v1.2.3...v1.2.4)Bump golang from 1.14 to 1.15.0
+  Bumps golang from 1.14 to 1.15.0.
```

Comment:
There is some issue with double entries and newlines on the last and second to last line there!

Also, this should be formatted slightly differently. Since this ends up in markdown, if you do not add bullet points, most of that content will render on one line, with little to separate the entries. I suggest using this as a target test case:

```
* Aggregated Dependabot Changelogs:
  * Bump github.com/klauspost/pgzip from 1.2.3 to 1.2.4
  * Bumps [github.com/klauspost/pgzip](https://github.com/klauspost/pgzip) from 1.2.3 to 1.2.4.
    - [Release notes](https://github.com/klauspost/pgzip/releases)
    - [Commits](klauspost/pgzip@v1.2.3...v1.2.4)
  * Bump golang from 1.14 to 1.15.0
```

Reply:
Oh wait, now I understand why it's double, it's because it appears like that in the commit message. Maybe just drop the title line then, since it is repeated in the body.

Reply:
So the output would be this:

```
* Aggregated Dependabot Changelogs:
  * Bumps [github.com/klauspost/pgzip](https://github.com/klauspost/pgzip) from 1.2.3 to 1.2.4.
    - [Release notes](https://github.com/klauspost/pgzip/releases)
    - [Commits](klauspost/pgzip@v1.2.3...v1.2.4)
  * Bumps golang from 1.14 to 1.15.0
```